### DISCUSSION TITLE
ColorPicker 控件的清空状态， 在 size 为 large, small 时，显示变形

### DISCUSSION THREAD

**Discussion by ekuku1984** [ant-design/ant-design#44256] [General]
https://codesandbox.io/s/colorpicker-test-xfj4nh?file=/src/index.js

在不同的 size 状态下， 清空状态的图显示大小没有变化， 导致在 small 时，显示变形， large 下， 虽然没有变形， 单是间距变大

![image](https://github.com/ant-design/ant-design/assets/6016942/a90ae9f1-734f-4d1e-bcec-17563278b370)

另外，清空值时， 返回的值是全透明的，而不是空值，不知道为什么会这么设计

**Comment by MadCcc** [comment]
> 另外，清空值时， 返回的值是全透明的，而不是空值，不知道为什么会这么设计

因为颜色没有空值，透明就是没有颜色。当然你也可以使用 `onClear` 来定制你期望的行为
